bonus part/fibonacci_continuous.py

```diff
@@ -1,49 +1,56 @@
-def fibonacci(x):
-    if x <= 0:
-        print("Incorrect input.")
-    elif x == 1:
-        return 0
-    elif x == 2:
-        return 1
-    else:
-        return fibonacci(x-1)+fibonacci(x-2)
+class FibonacciSave:

```

Comment:
@RomanKlasnyy please remove this class and pass values as methods arguments. Now, this class is similar to global variables that are not the best solution.